### ISSUE TITLE
What do about Sam's Fetch partner

### ISSUE THREAD

**Issue by rafd** [fish-bucket/ops#2]
Background:
Sam has been working on Fetch with a partner from work ("Other Simon")
The current plan for Fishbucket doesn't have a way to deal with "external partners" on projects.
We may run into similar problems in the future.

Some potential solutions:

**(1) Make "Other Simon" a Partner**

**(2) Make "Other Simon" an Investor**
"Other Simon" would invest in Fishbucket. All Fetch IP would belong to Fishbucket, but "Other Simon" would still be incentivized to help w/ Fetch b/c of position in Fishbucket.

**(3) Spin out Fetch as a Corporation**
 w/ ownership between Fishbucket and "Other Simon"

**(4) A Sales/Referral Agreement**
"Other Simon" could receive a cut of sales that he refers.

(5) ???

**Comment by rafd** [comment]
@sjuraschka since you are closest to this issue, do you mind taking the "R" (and follow the advice process).

**Comment by rafd** [comment]
Also worth keeping in mind that Fishbucket isn't a real thing yet, may end up dying.

...but, still worth knowing what the potential future paths are for this issue (better now than in 4 months after a lot more work has been put into Fetch)

**Comment by sjuraschka** [comment]
@rafd yes for sure, I can take the R on this. It has been something I've been thinking about a lot since we first met. I pitched it to him briefly after the first time we met - and he thought it was interesting, but he could also just be humouring me. I think I actually need to educate him on other companies that are building multiple properties, multiple founders, using alternate funding solutions.
